aiter/ops/triton/mha.py

```diff
@@ -3048,6 +3113,8 @@ def flash_attn_fp8_func(
     deterministic=False,
     return_lse=False,
     return_attn_probs=False,
+    fused_backward=False,
+    onekernel_backward=False,
```

Comment:
Remove these args from the top level API. Instead add a setter function which sets a global variable to control which kernel to use. See the MoE code for example https://github.com/ROCm/aiter/blob/main/aiter/ops/triton/moe_op.py#L18